apps/test/unit/p5lab/spritelab/CoreLibraryTest.js

```diff
@@ -169,6 +179,38 @@ describe('SpriteLab Core Library', () => {
     });
   });
 
+  describe('Sprite limit', () => {
+    // The number of sprites created is limited by MAX_NUM_SPRITES
+    it(`caps at ${MAX_NUM_SPRITES} sprites`, () => {
+      for (let i = 0; i < 50000; i++) {
+        coreLibrary.addSprite();
+      }
+      expect(coreLibrary.getNumberOfSprites()).to.equal(MAX_NUM_SPRITES);
+    });
+
+    // When number of sprites is reaches MAX_NUM_SPRITE_WARNING,
+    // an additional addSprite call will result in displayWorkspaceAlert being dispatched
+    // Note that MAX_NUM_SPRITES = MAX_NUM_SPRITE_WARNING + 1
+    it(`When ${MAX_NUM_SPRITE_WARNING} sprites is reached, workspace alert is dispatched`, () => {
+      const stubbedDispatch = stub();
+      stub(redux, 'getStore').returns({
+        dispatch: stubbedDispatch
+      });
+      for (let i = 0; i < MAX_NUM_SPRITES; i++) {
+        coreLibrary.addSprite();
+      }
+      expect(stubbedDispatch).to.have.been.calledWith(
```

Comment:
NIT: should `calledWith` here be `calledOnceWith`? ([sinon-chai cheat sheet](https://www.chaijs.com/plugins/sinon-chai/) if it's helpful)

Reply:
Thanks! Made the update.

Reply:
Also added 2 more unit tests.